tests/lustre/typechecking/shouldFail/test_homeomorphic_exn.lus

```diff
@@ -0,0 +1,6 @@
+type t = [int, int, bool];
+
+node n2 (x, y: t) returns (z: [int, int, bool]);
+let
+z = x + y; -- this should fail, but doesn't
+tel.
```

Comment:
I’m getting the following unexpected error (see more comments on Slack):
> <Error> Error opening input file 'tests/lustre/typechecking/shouldFail/test_homeomorphic_exn.lus': Not_found

Reply:
Fixed this.